pkg/releasepayload/controller/helpers.go

```diff
@@ -0,0 +1,58 @@
+package controller
+
+import (
+	"fmt"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// GetNamespaceAndName returns the namespace and name from the runtime.Object
+// or an error if either is missing.
+func GetNamespaceAndName(object runtime.Object) (namespace, name string, err error) {
```

Comment:
where do you use this?  I find it rarely makes code easier to read to have these helpers.

Reply:
It's not currently used anywhere.  I added this function to process a `runtime.Object` via an `meta.NewAccessor()`.  I wasnt quite sure of an appropriate pattern for using the `Accessor` properly and I chose to add a couple of functions here...